clojure/config.ini

```diff
@@ -5286,6 +5286,10 @@ name = Arthur Barroso
 filter = (clojure|Clojure|\(def |\(defn-? )
 twitter = 32a852
 
+[https://kozieiev.com/tags/clojure/]
```

Comment:
you need to add RSS feed. I see that you have it, but not sure if you have per category. If don't, then add the top-level RSS feed, and add filter

Reply:
@alexott , fixed